src/main/webapp/login-status.js

```diff
@@ -0,0 +1,14 @@
+// Get signed in user info
+function onSignIn(googleUser) {
+  const profile = googleUser.getBasicProfile();
+  const idToken = googleUser.getAuthResponse().id_token;
+  console.log('Name: ' + profile.getName());
+  console.log('Image URL: ' + profile.getImageUrl());
+  console.log('Email: ' + profile.getEmail());
+}
+
+// Sign user out
+function signOut() {
+  const auth2 = gapi.auth2.getAuthInstance();
+  auth2.signOut();
+}
```

Comment:
After I sign out, I get redirected to the home page. Would it be possible to get redirected back to the page I was already on?

Reply:
Fixed!

Reply:
Thanks! Nit: would it be possible to also make the page refresh? When I click the sign-out button it kind of feels like nothing happens, and I have to scroll back up and notice that the sign-in button now says "sign in" instead of "signed in". This is really a nit though